src/tests/test_bootstrap_in_place.py

```diff
@@ -39,6 +39,23 @@
 INSTALLER_GATHER_DEBUG_STDERR = os.path.join(INSTALLER_GATHER_DIR, "gather.stderr.log")
 
 
+# TODO: Find a more elegant solution than hard-coding RHCOS URLs... Assisted
+# service maintains an up-to-date list over at:
+# https://github.com/openshift/assisted-service/blob/133ed9cf68e36590f19901d71813399bb86b165b/data/default_os_images.json#L1-L44  # noqa: E501,B950
+# maybe we can consume it somehow here? Although that would create a dependency
+# on assisted-service which we don't want for this job.
+class RHCOSImages:
+    _OPENSHIFT_RHCOS_MIRROR = "https://mirror.openshift.com/pub/openshift-v4/dependencies/rhcos"
```

Comment:
it can be extracted from the payload

./openshift-install coreos print-stream-json | grep x86_64.iso